core/v0/api/core.json

```diff
@@ -2344,14 +2338,15 @@
                     },
                     "agent": {
                         "description": "The person who fulfills the order",
-                        "allOf": [
-                            {
```

Comment:
Can you please create a schema called `Agent` with all these attributes and add a `$ref` here?. Adding a `person` and `contact` property to the `Agent` schema breaks compatibility with existing implementations.

Reply:
Discussed on the call. Decided push the current change.

Reply:
Discussed again. Decided to create Agent schema with allOf from Person and Contact, make cred mandatory.

Reply:
Done.